usb.c

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 Google Inc.  All rights reserved.
+// Copyright 2018 the gousb Authors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <libusb.h>
+
+void gousb_set_debug(libusb_context *ctx, int lvl) {
+    // TODO(sebek): remove libusb_debug entirely in 2.1 or 3.0,
+    // require libusb >= 1.0.22. Accept a Go-typed levels instead
```

Comment:
Why does this say require libusb >= 1.0.22, but below you compare with 0x01000106?

Reply:
The documentation for libusb_set_option (http://libusb.sourceforge.net/api-1.0/group__libusb__lib.html) says:

Since version 1.0.22, LIBUSB_API_VERSION >= 0x01000106

Added that to the comment.